client/app/scripts/superdesk-archive/module.js

```diff
@@ -172,31 +172,14 @@ define([
                 })
                 .activity('duplicate-content', {
                     label: gettext('Duplicate'),
-                    icon: 'copy',
-                    //monitor: true,
-                    /*
-                    controller: ['api', 'data', 'desks', '$rootScope', function(api, data, desks, $rootScope) {
-                        api
-                            .save('duplicate', {}, {desk: desks.getCurrentDeskId()}, data.item)
-                            .then(function(archiveItem) {
-                                data.item.task_id = archiveItem.task_id;
-                                data.item.created = archiveItem._created;
-                                $rootScope.$broadcast('item:duplicate');
-                            }, function(response) {
-                                data.item.error = response;
-                            })
-                        ['finally'](function() {
-                            data.item.actioning.archiveContent = false;
-                        });
-                    }],
-                    */
+                    icon: 'archive',
```

Comment:
Why are we replacing copy icon with archive icon here? In personal desk the requirement was to display copy icon.